index.php

```diff
@@ -84,7 +75,7 @@ function getinterval()
     <div class="col-md-12">
     <div class="box" id="queries-over-time">
         <div class="box-header with-border">
-          <h3 class="box-title">Total queries over last <?php echo getinterval(); ?> hours</h3>
+          <h3 class="box-title">Total queries over last <span class="maxlogage-interval">24</span> hours</h3>
```

Comment:
Why is 24h still hardcoded?

Reply:
We need something to show before javascript replaces it (and 24 is the default value).